src/MongoDB.Driver.Legacy/MongoDB.Driver.Legacy.csproj

```diff
@@ -15,11 +15,12 @@
     <Copyright>Copyright © 2010-present MongoDB Inc.</Copyright>
     <Description>Legacy MongoDB supported driver for MongoDB.  See http://www.mongodb.org/display/DOCS/CSharp+Language+Center for more details.</Description>
     <Authors>rstam;craiggwilson</Authors>
+    <PackageId>mongocsharpdriver</PackageId>
```

Comment:
Normally the PackageId is the same as (and defaults to) the Assembly name.

But in the case of the Legacy package we are using the original package name that was used before 2.0 came out.